java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -857,6 +857,84 @@ public static Table readCSV(Schema schema, CSVOptions opts, HostMemoryBuffer buf
         opts.getFalseValues()));
   }
 
+  private static native void writeCSVToFile(long table,
+                                            String[] columnNames,
+                                            boolean includeHeader,
+                                            String rowDelimiter,
+                                            byte fieldDelimiter,
+                                            String nullValue,
+                                            String trueValue,
+                                            String falseValue,
+                                            String outputPath) throws CudfException;
+
+  public void writeCSVToFile(CSVWriterOptions options, String outputPath)
+  {
```

Comment:
I assume that you didn't run formatting on this file.
```suggestion
  public void writeCSVToFile(CSVWriterOptions options, String outputPath) {
```

Reply:
Oh. What formatter are you running on Java files?

Reply:
TIL that I should be using `pre-commit run --all-files` instead of the `clang-format` script.